src/test/java/com/wikia/webdriver/testcases/mobilewikitests/CommentsTests.java

```diff
@@ -64,46 +64,46 @@ public void mercury_comments_containsAvatarUsernameTimestampAndContent() {
         "Comments are collapsed"
     );
 
-    PageObjectLogging.log(
+    Log.log(
         "Comments list",
         "is expanded",
         true
     );
 
     boolean result = comments.getNumberOfCommentsPerPage() == NUMBER_OF_COMMENTS_PER_PAGE;
-    PageObjectLogging.log(
+    Log.log(
         "Number of comments per page",
         "is correct",
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "is correct" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)